tests/Scientist/Domain/MarsScientist/MarsScientistTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+declare(strict_types=1);
+
+namespace App\Tests\Scientist\Domain\MarsScientist;
+
+use App\DomainModel\MarsScientist;
+use App\Exception\ScientistIsAliveException;
+use PHPUnit\Framework\TestCase;
+
+class MarsScientistTest extends TestCase
+{
+    private MarsScientist $scientist;
+
+    protected function setUp(): void
+    {
+        $this->scientist = new MarsScientist(
+            (int) null,
+            'Adam',
+            'Jensen',
+            (string) null,
+            ['test'],
+            ['test'],
+            ['test']
+        );
+    }
+
+    /** @test */
+    public function test_creating_new_scientist(): void
+    {
+        $newScientist = MarsScientist::createNewScientist('Adam', 'Jensen');
+
+        $this->assertSame($this->scientist->getApikey(), $newScientist->getApikey());
+    }
+
+    /** @test */
+    public function test_marking_as_missing(): void
+    {
+        $this->scientist->markAsMissing();
+
+        $this->assertTrue($this->scientist->isMissing());
+    }
+
+    /** @test */
```

Comment:
Jak cos to nie trzeba dodawac tych adnotacji

Reply:
Wiem, ale czasami lubialo rzucic bledem ze nie ma testow bez niej, albo jak metoda sie nie zaczynala "test"